pkg/controllers/pushsecret/pushsecret_controller.go

```diff
@@ -243,7 +248,12 @@ func (r *Reconciler) PushSecretToProviders(ctx context.Context, stores map[esapi
 		}
 		client, err := mgr.Get(ctx, storeRef, ps.GetNamespace(), nil)
 		if err != nil {
-			return out, fmt.Errorf("could not get secrets client for store %v: %w", store.GetName(), err)
+			// if store is not handled by this controller instance
+			if err.Error() == "can not reference unmanaged store" {
+				return out, nil
```

Comment:
Same here.

Probably better to just `continue` if it is an unmanaged SecretStore.